submissions/emmanuelEsuong/css/styles-mediaQuery.css

```diff
@@ -0,0 +1,264 @@
+    #regis {
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 40px;
+        font-weight: bold;
+        color: rgb(2, 209, 2);
+    }
+    #rg, #rgs, #gt {
+        text-align: center;
+        margin-left: 7.5cm;
+        padding-bottom: 0px;
+        padding-top: 0px;
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 28px;
+        text-align: center;
+    }
+    #signin{
+        color:rgb(2, 209, 2);
+    }
+    #supT {
+        height: 22.5cm;
+    }
+    footer {
+        height: 5.3cm;
+    }
+    #icons {
+        margin-right: 1.5cm;
+    }
+    #copyright {
+        margin-left: 6cm;
+        margin-top: 1cm;
+    }
+}
+/* media-query (mobile)*/
+@media screen and (min-width: 320px) and (max-width: 480px) {
+    /* top navigation */
+    .topnav {
+        overflow: hidden;
+        background-color: rgb(2, 209, 2);
+        text-align: center;
+        font-size: 35px;
+    }
+    .topnav #logo {
+        margin-left: 0.7cm;
+        margin-top: 0.5cm;
+    }
+    .topnav a {
+        float: left;
+        padding: 10px 20px;
+        text-decoration: none;
+        color: white;
+    }
+    .topnav-right {
+        float: left;
+        margin-top: 0cm;
+        margin-right: 19cm;
+        margin-left: 0.5cm;
+        width: 300px;    
+    }
+    /* signUp page */
+    /* formfield */
+    #s-form {
+        border-radius: 25px;
+        border: solid 1px;
+        padding: 35px 20px 20px 20px;
+        border: 2px 2px 2px 2px
+    }    
+    /* signupform */
+    #form {
+        overflow: hidden;
+        margin-top: 10px;
+        float: right;
+        margin-right: 3.5cm;
+    }
+    #form input {
+        width: 650px;
+        height: 1cm;
+        padding: 10px;
+        border-radius: 10px;
+        border: solid 1px;
+    }
+    #table, #form {
+        display: inline-block;
+    }
+    #table {
+        margin-right: 2cm;
+        float: right;
+        width: 1100px;
+    }
+    #form #terms {
+        width: 20px;
+        float: left;
+        margin-left: 3cm;
+    }
+    #termsLabel {
+        color: rgb(2, 209, 2);
+        font-weight: bold;
+        margin-left: 10px;
+        padding-top: 40px;
+    }
+    button {
+        border-radius: 3px;
+        border: 0px;
+        margin-left: 150px;
+        padding: 7px;
+        padding-left: 20px;
+        padding-right: 20px;
+        background-color: rgb(2, 209, 2);
+        font-weight: bold;
+        color: white;
+    }  
+    ul {
+        list-style-type: none;
+        padding: 0px;
+        overflow: hidden;
+    }
+        /* registration info */
+    #regis {
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 45px;
+        font-weight: bold;
+        color: rgb(2, 209, 2);
+    }
+    #rg, #rgs, #gt {
+        text-align: center;
+        margin-left: 7.5cm;
+        padding-bottom: 0px;
+        padding-top: 0px;
+        font-family: Arial, Helvetica, sans-serif;
```

Comment:
Why not declare the font-family for the whole page instead of just here?

Reply:
Updated.